source/config/__init__.py

```diff
@@ -64,7 +64,13 @@ def isInstalledCopy():
 
 def getInstalledUserConfigPath():
 	try:
-		return os.path.join(shlobj.SHGetFolderPath(0, shlobj.CSIDL_APPDATA), "nvda")
+		k = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, NVDA_REGKEY)
+		configInLocalAppData = bool(_winreg.QueryValueEx(k, u"configInLocalAppData")[0])

```

Comment:
Rather than hard code  the "configInLocalAppData" string here and in `installer.py` could you move it to a shared constant in the same way that `NVDA_REGKEY` is. Please also add a docstring to explain what this key is for and what the valid values are. In addition, referencing the github issue where it is discussed would likely be helpful.